src/main/java/duke/command/ModifyCommand.java

```diff
@@ -0,0 +1,65 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Command child class that lists tasks or marks task as done/undone.
+ */
+public class ModifyCommand extends Command {
+    /**
+     * Types of commands that can be executed by this class.
+     */
+    public enum CommandType {
```

Comment:
Should all subclasses of command have a CommandType attribute similar to this ModifyCommand class? Or perhaps List, Undone and Done commands should be their own separate Commands?